backend/routes/boards.js

```diff
@@ -51,28 +47,38 @@ router.patch('/edit-board', authJWT, async (req, res) => {
   const { id, name } = req.body;
   const { id: userId } = req.user;
 
-  if (!id) {
+  if (id === undefined) {
     return res
       .status(400)
       .json({ id: null, error_message: boardErrorMessages.MISSING_ID });
   }
 
-  if (!name) {
+  if (name === undefined) {
     return res
       .status(400)
       .json({ id: null, error_message: boardErrorMessages.MISSING_NAME });
   }
 
-  if (!userId) {
+  const userPermissions = await getPermissions(userId, id);
```

Comment:
i sud lang sad nato ug try catch guro, rather than mag check pa ta if undefined aron less ang chance maka server error ta.

Reply:
Maybe sa get permissions muthrow nalang gihapon ta ug error? aron consistent atong way sa pag call sa functions sud sa model

Reply:
okay, will work on that sir